dockerfiles/install_common_deps.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+DEBIAN_FRONTEND=noninteractive
+apt-get update && apt-get install -y --no-install-recommends \
+        wget \
+        zip \
+        ca-certificates \
+        build-essential \
+        curl \
+        libcurl4-openssl-dev \
+        libssl-dev \
+        python3-dev \
+        git
+
+# Dependencies: conda
+wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-4.5.11-Linux-x86_64.sh -O ~/miniconda.sh --no-check-certificate && /bin/bash ~/miniconda.sh -b -p /opt/miniconda
+rm ~/miniconda.sh
+/opt/miniconda/bin/conda clean -tipsy
+
+conda install -y python=3.6 numpy
+conda clean -aqy
+rm -rf /opt/miniconda/pkgs
+find / -type d -name __pycache__ -prune -exec rm -rf {};
```

Comment:
why is this find/prune command needed?

Reply:
It was originally included as a requirement from the Azure folks. It seems like it's clearing the conda installation from the python cache (for size reduction?). Any insight @manashgoswami?

Reply:
Removed...